openpype/hosts/houdini/plugins/create/create_bgeo.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating pointcache bgeo files."""
+from openpype.hosts.houdini.api import plugin
+from openpype.pipeline import CreatedInstance
+from openpype.lib import EnumDef
+
+
+class CreateBGEO(plugin.HoudiniCreator):
+    """BGEO pointcache creator."""
+    identifier = "io.openpype.creators.houdini.bgeo"
+    label = "BGEO PointCache"
+    family = "pointcache"
+    icon = "gears"
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        import hou
+
+        instance_data.pop("active", None)
+
+        instance_data.update({"node_type": "geometry"})
+
+        if not instance_data.get("families"):
```

Comment:
This could be done with:
```py
instance_data.setdefault("families", [])
```

Reply:
Why are the families filled in creation? Wouldn't be better to handle that in publishing?

Reply:
> Wouldn't be better to handle that in publishing?

I agree that it would make much more sense - except that _then_ you can't have attribute definitions for the new publisher exposed to those families. It's an unfortunate downside to the design of the new publisher.

Reply:
That would not be there even now.

Reply:
It would be, right? Since it's added on create and thus is present in the instance directly after? Or doesn't it support `families` to begin with? :)

Reply:
I've moved the families to collector as there is no need to have it set by creator because there are no additional options set by plugins determined by those families.

It should be backwards compatible with instances created with previous creators.

I agree with storing of data on node, we should address it in another PR